examples/getting-started-app/README.md

```diff
@@ -640,6 +655,9 @@ As a result, we'll get two replica sets, with one Tarantool instance in each.
 
 ![Two replica sets](./images/two-replicasets.png)
 
+Now we have two replicasetss realizing their roles, but `vshard` is not running yet.
+Press the button `Bootstrap vshard` on the admin's page.
```

Comment:
Now we have two replica sets implementing their roles, but `vshard` is not running yet.
Press the button **Bootstrap vshard** on **Cluster** tab.